Gems/Terrain/Assets/Shaders/Terrain/ClipmapComputeHelpers.azsli

```diff
@@ -33,9 +32,10 @@ float2 GetCurrentClipmapCenter(uint clipmapLevel)
     return TerrainSrg::m_clipmapData.m_clipmapCenters[clipmapLevel].zw;
 }
 
+// Get the world position at the pixel position in a clipmap.
 float2 GetWorldPosition(float2 clipmapCenter, uint2 pixelPosition, uint clipmapLevel)
```

Comment:
Suggestion: it would also be good to comment on what "clipmapCenter" is, how it's the conceptual center, not the center of the texture.

Reply:
Ok. Later we will remove clipmap center to use ClipmapBounds class.